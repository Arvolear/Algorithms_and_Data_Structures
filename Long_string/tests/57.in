63672514814281
6348174059214039765