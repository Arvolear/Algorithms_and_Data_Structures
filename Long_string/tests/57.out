6348237731728854046
6348110386699225484
404204206828940297044386723883965
99700